### PR TITLE
Promote SupplyChain and EID/UID objects to the mainline specification

### PR DESCRIPTION
**Pull request by simontrasler**
As defined in https://github.com/InteractiveAdvertisingBureau/openrtb2.x/blob/main/2.6.md.

### CODE REVIEW

File: openrtb-core/src/main/protobuf/openrtb.proto [hunk 1]
```diff
@@ -154,6 +154,86 @@ message BidRequest {
...
+        extensions 100 to 9999;
+      }
+
+      // Array of SupplyChainNode objects in the order of the chain. In a
+      // complete supply chain, the first node represents the initial
+      // advertising system and seller ID involved in the transaction, i.e. the
+      // owner of the site, app, or other medium. In an incomplete supply chain,
+      // it represents the first known node. The last node represents the entity
+      // sending this bid request.
+      repeated SupplyChainNode nodes = 3;
```
Comment (simontrasler) [inline openrtb-core/src/main/protobuf/openrtb.proto]: Note, Google has the indices for `ver` and `nodes` the other way around. I will discuss with them.

Comment (simontrasler) [reply openrtb-core/src/main/protobuf/openrtb.proto]: Decided to switch to the new indices during the promotion. The `SupplyChainNode` object can be re-used entirely between the two implementations.

**Review by cindyhere33 — APPROVED** [review]

**Comment by cindyhere33** [comment]
The adcoms-proto submodule is being put back to an older commit, think that needs to be removed from this PR. Approved it before noticing it.

**Comment by simontrasler** [comment]
> The adcoms-proto submodule is being put back to an older commit, think that needs to be removed from this PR. Approved it before noticing it.

Good catch, fixed.

**Review by cindyhere33 — APPROVED** [review]

**Pull request merged**